play-services-base-core-ui/src/main/res/values-it/strings.xml

```diff
@@ -9,4 +9,6 @@
 
     <string name="list_no_item_none">Nessuna</string>
     <string name="list_item_see_all">Mostra tutte</string>
+
+    <string name="open_app">Aprire</string>
```

Comment:
Please don't add this. The translation is wrong and the missing string has been addressed in #1457 (waiting to be merged).

Reply:
Thank you for the help! My Italian is not the best, so I should have seen that one coming.
Tried to cancel any changes done to this file in my commit, and since it is my first time doing something serious with git, I seem to have just pulled a commit to remove the file altogether. Could you please help me fix it now?

Reply:
I think that the simplest way in your scenario is to make another commit that removes the lines you added to the file.
Otherwise you could do a `git checkout <commit sha> -- <path/to/file>` to replace the current contents of the file with the contents of the file in that specified commit.

Btw, if you're new to Git I'd recommend you to follow [this course on Pluralsight](https://www.pluralsight.com/courses/how-git-works) (there is a 10-day free trial), it really helped me a lot!

Reply:
Oh, sure — I forgot I was supposed to actually remove my string from the file. Got distracted by my mistake. The line is no longer there in my pull request, thanks!

Reply:
Sorry if I sound pedantic, but it would be best if you could also remove the extra new line added to the Italian strings file, so that this PR won't conflict with #1457 when any of the two gets merged. Thanks :)

Reply:
No problem! :D